www/source/blog/2018-06-22-base-plans-refresh.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Core Plans and Certain Linux Kernels
+date: 2018-06-22
+author: Nell Shamrell-Harrington
+tags: Core Plans
+category: update
+classes: body-article
+---
+
+Greetings, fellow Habicats!
+
+This past Tuesday we completed a refresh [of the Habitat Core Plans](https://www.habitat.sh/blog/2018/06/base-plans-refresh/).
+
+While we expected some dependency resolution errors to happen as a result of the refresh, another issue came up which we did not anticipate.
+
+## What Happened?
+
+Part of the major base plans refresh was upgrading [core/glibc](https://github.com/habitat-sh/core-plans/tree/master/glibc) from 2.22 to 2.27.
+
+[GNU glibc](https://www.gnu.org/software/libc/) is the interface to the Linux kernel. Most if not all Linux core plans (and Habitat core components for Linux) use glibc. We did not realize before the refresh that glibc 2.26 and up require the kernel headers from Linux kernel 3.2 or later. This was partly due to the length of time since the last base plans refresh (over a year) - so many things had to be upgraded that we missed the change in required kernel.
+
+This meant that after we completed the base plans refresh anyone who attempted to use any Habitat components or core plans on a system running a Linux kernel lower than 3.2 starting receiving failures.
```

Comment:
It might be good to find a way to move this up. The explanation is great, butfor your reader it might be good to have the problem statement up front as one of the first sentences of this message.

Reply:
Ha! I just see that I left this tab open too long and this was merged a while ago.  Ignore me! :D 

Overall, thanks for getting this blog post out!